lib/AL_USDMaya/AL/usdmaya/nodes/ProxyShape.h

```diff
@@ -231,6 +231,7 @@ struct FindLockedPrimsLogic
 };
 
 typedef const HierarchyIterationLogic*  HierarchyIterationLogics[3];
+typedef TfHashMap<SdfPath, MString, SdfPath::Hash > _PrimPathToDagPath;
```

Comment:
please remove the '_' - it may be in Pixars coding scheme, but it's not a part of the C++ standard